apps/server/src/infra/authorization-client/authorization-client.adapter.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable, InternalServerErrorException } from '@nestjs/common';
+import { ErrorUtils } from '@src/core/error/utils';
+import { AuthorizationApi, AuthorizationBodyParams, AuthorizedReponse } from './authorization-api-client';
+
+@Injectable()
+export class AuthorizationClientAdapter {
```

Comment:
@SevenWaysDP @bergatco würde es nicht ausreichen wenn wir hier von AuthorizationClient sprechen? Ein Client ist zwingendermaßen immer ein Adapter. Ich finde durch Adapter erhöht sich nicht die Aussagekraft. Nur der Name wird länger.

Reply:
Ich denke, Du hast recht. Der `AuthorizationClient` verwendet dann die `AuthorizationApi` um mit dem Backend zu reden.
Sollte dann nicht auch `apps/server/src/infra/authorization-client/authorization-api-client` in `apps/server/src/infra/authorization-client/authorization-api` umbenant werden?

Reply:
In Absprache mit @CeEv wurde entschieden, das aktuelle Namensschema beizubehalten.